src/components/Product.tsx

```diff
@@ -5,116 +5,116 @@ interface ProductProps {}
 
 export const Product: React.FC<ProductProps> = (props) => {
   return (
-    <Row>
-      <CheckContainer>
+    <ProductContainer>
+      <div>
         <input type="checkbox" />
-      </CheckContainer>
-      <ImageColumn>
-        <ProductImage />
-        <ButtonRow>
-          <Button>
-            <ButtonImage src={process.env.PUBLIC_URL + '/decrement.svg'} />
-          </Button>
-          <Count>
-            <CountText>1</CountText>
-          </Count>
-          <Button>
-            <ButtonImage src={process.env.PUBLIC_URL + '/increment.svg'} />
-          </Button>
-        </ButtonRow>
-      </ImageColumn>
-      <ProdcutInformationContainer>
-        <Row>
-          <ProuductTitle>
+      </div>
+      <div className="product-flex-column product-image-container">
+        <img className="product-main-image" />
+        <div className="product-flex-row image-button-container">
+          <button className="image-button">
+            <img src={process.env.PUBLIC_URL + '/decrement.svg'} />
+          </button>
+          <div className="product-count">
+            <div>1</div>
+          </div>
+          <button className="image-button">
+            <img src={process.env.PUBLIC_URL + '/increment.svg'} />
+          </button>
+        </div>
+      </div>
+      <div className="product-flex-column product-information-container">
+        <div className="product-flex-row">
+          <div className="product-title">
             productnamethisisneverthatdfasdfasdfascasdfascsd
-          </ProuductTitle>
-          <DeleteButton outlined={true} active={false}>
+          </div>
+          <CustomButton className="delete-button" outlined active={false}>
             삭제
-          </DeleteButton>
-        </Row>
-        <MaxRow>
-          <ProductOptionStatus>
+          </CustomButton>
+        </div>
+        <div className="product-flex-row product-max-height">
+          <div className="product-flex-column product-option-status">
```

Comment:
`product-flex-row product-max-height`여기는 className에 css 효과를 쓰기보다는 어떤 역할을 하는지 쓰는게 더 나을 것 같아요. `product-flex-row` `product-max-height`에 해당하는 스타일이 있어서 구별하신 것 같은데 그러면 `product-option-status-block` 라던지 `product-option-status-wrapper`라던지 감싸고 있는 역할인걸 명시해준후에 쓰면 조을거같습니다.

`flex-row,column `겹치는 스타일들때문에 중복줄일라고 따로 클래스명만들어서 하신것같은데 그렇게 해도 좋은데 뭔가 좀 보기 어려운거같아요..!

```suggestion
        <div className="product-option-status-wrapper">
          <div className="product-option-status product-flex-column">
```